### PR TITLE
fn user added to runtime image

### PR DESCRIPTION
**Pull request by nishalad95**

### CODE REVIEW

**Comment by rdallman** [comment]
thanks for starting on this. there's likely quite a few more to update, I think that the base image may catch quite a few of them (go:dev appears to not, the python3.6 here neither, java seems ok - wrt 'official' lang support)

**Comment by nishalad95** [comment]
removed fn user from node/dev as the CLI uses fnproject/node as the runtime image

File: node/dev/Dockerfile [hunk 1]
```diff
@@ -7,3 +7,4 @@ FROM node:alpine
 #   && apk add git build-base nodejs-dev@edge nodejs@edge \
 #   && npm install -g npm \
 #   && rm -rf /var/cache/apk/*
+
```
Comment (templecloud) [inline node/dev/Dockerfile]: Is this as accidental change? If so can we tidy it up? Just so it is super clear what we are changing here.

Comment (templecloud) [reply node/dev/Dockerfile]: Maybe even clear out the comment in the file if they are no longer needed?

File: node/Dockerfile [hunk 2]
```diff
@@ -7,3 +7,5 @@ FROM node:alpine
 #   && apk add nodejs@edge \
 #   && npm uninstall -g npm \
 #   && rm -rf /var/cache/apk/*
+
+RUN addgroup -g 1000 -S fn && adduser -S -u 1000 -G fn fn
```
Comment (templecloud) [inline node/Dockerfile:11]: I understand what this is supposed to do and it looks good. Did we 'test' it by spinning up a container, exec-ing in and checking it? Did we test a standard deploy and invoke?

File: node/dev/Dockerfile (hunk 1 above)
Comment (nishalad95) [reply node/dev/Dockerfile]: yup was accidental - updated

File: node/Dockerfile (hunk 2 above)
Comment (nishalad95) [reply node/Dockerfile:11]: i think this is the second round of testing

**Pull request merged**